src/services/dissolution/dissolution.service.ts

```diff
@@ -50,6 +52,18 @@ export default class DissolutionService {
     return this.paymentMapper.mapToPaymentSummary(dissolutionGetPaymentUIData)
   }
 
+  public async patchDissolutionPaymentData(
```

Comment:
i would give this method a more specific name i.e. patch pay by account data.You can hardcode the PaymentStatus rather than passing a param in that case.